assets/app/views/_project-nav.html

```diff
@@ -1,42 +1,36 @@
-<div class="navbar-project">
-  <div class="navbar-project-menu">
```

Comment:
there were several styles referencing `.navbar-project-menu` in `_core.less`... at least remove them, and might be worth checking with @sg00dwin about the reason for some of them to make sure we don't break something